src/Silverback.Integration/Messaging/Configuration/ConsumerEndpointBuilderMessageValidationExtensions.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2020 Sergio Aquilini
+// This code is licensed under MIT license (see LICENSE file for details)
+
+using Silverback.Messaging.Validation;
+using Silverback.Util;
+
+namespace Silverback.Messaging.Configuration
+{
+    /// <summary>
+    ///     Adds the <c>ValidateMessage</c> method to the
+    ///     <see cref="ConsumerEndpointBuilder{TEndpoint,TBuilder}" />.
+    /// </summary>
+    public static class ConsumerEndpointBuilderMessageValidationExtensions
+    {
+        /// <summary>
+        ///     Specifies which validation mode has to be applied to the messages.
+        /// </summary>
+        /// <typeparam name="TBuilder">
+        ///     The actual builder type.
+        /// </typeparam>
+        /// <param name="endpointBuilder">
+        ///     The endpoint builder.
+        /// </param>
+        /// <param name="mode">
+        ///     The validation mode (see <see cref="ValidationMode"/>.
+        /// </param>
+        /// <returns>
+        ///     The endpoint builder so that additional calls can be chained.
+        /// </returns>
+        public static TBuilder ValidateMessage<TBuilder>(
```

Comment:
I would maybe create two distinct methods though:
* `builder.ValidateMessage(throwException: bool)`
* `builder.DisableMessageValidation()`

I think the API would be a bit nicer. What do you think? I'm open to other point of views since this is not really a black/white thing.